src/Data/Swagger/Internal/ParamSchema.hs

```diff
@@ -218,6 +219,11 @@ instance ToParamSchema T.Text where
 instance ToParamSchema TL.Text where
   toParamSchema _ = toParamSchema (Proxy :: Proxy String)
 
+instance ToParamSchema Version where
+  toParamSchema _ = mempty
+    & type_ .~ SwaggerString
+    & pattern ?~ "^[\\d+.]*[\\-[\\w]+]*$"
```

Comment:
Can we pretend there aren't *tags*? (E.g. `Cabal`'s version doesn't have them). Also e.g. `QuickCheck` pretends here aren't tags (http://hackage.haskell.org/package/QuickCheck-2.11.3/docs/src/Test-QuickCheck-Arbitrary.html#line-950) too.

Also the digit part is 

```
\d+(\.\d+)*
```

I.e.

- `0.1.` isn't valid version
- and dot `.` have to be escaped in regexp.

Reply:
I'd rather not have a pattern than one that fails on valid values, personally.

Reply:
I dropped the tags part of the schema and fixed it to be what you said :) thanks!